privacyidea/static/components/user/views/user.html

```diff
@@ -47,6 +47,25 @@
                                 translate>Edit realms</a>
                     </li>
                     <li translate>total users: {{ usercount }}</li>
+                    <hr>
+                    <li role="presentation"
+                        ng-show="($state.includes('user.list'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/webui/index.html#users'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about Users</span></a></li>
+                    <li role="presentation"
+                        ng-show="($state.includes('user.add'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/webui/manage_users.html#users'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about manage Users</span></a></li>
+                    <li role="presentation"
+                        ng-show="($state.includes('user.details'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/webui/user_details.html#users'}}">
```

Comment:
the `#users` anchor does not exist on that page

Reply:
done in b5be0d720e77db9fe1eab889e9384eb1e4c7a521.